### DISCUSSION TITLE
Support for incremental backups

### DISCUSSION THREAD

**Discussion by VaZark** [Archmonger/django-dbbackup#565] [Feature Requests]
### Feature description

With Postgres 17, almost all official DBs (except SQLite) offer some form of incremental backup functionality.

Regular incremental backups + occasional full backups will be easier to manage in terms of task intensity and resiliency. Based on Barman's blog, recovery times dropped from 78 minutes to just 4. ([link](https://www.enterprisedb.com/blog/why-postgresql-17s-incremental-backup-feature-game-changer))

**Comment by johnthagen** [comment]
Agree that incremental backup would be a really powerful feature.
